ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/tools/osb/mbean/ObjStoreBrowser.java

```diff
@@ -328,9 +341,26 @@ public ObjStoreBrowser() {
      * In other words, this class should be considered as a singleton class. In fact, if two ObjStoreBrowser instances
      * are created, they may interfere with each other.</p>
      */
-   public ObjStoreBrowser(String logDir) {
-        init(logDir);
-        this.objStoreBrowserMBeanName = arjPropertyManager.getObjectStoreEnvironmentBean().getJmxToolingMBeanName();
+     // Deprecated, use the getInstance method instead since this is a singleton instance
+     // if needed you can set the logDir as 'arjPropertyManager.getObjectStoreEnvironmentBean().setObjectStoreDir(logDir)'
+    @Deprecated(forRemoval = true)
+     private ObjStoreBrowser(String logDir) {
```

Comment:
I think this should still be public as moving it to private is effectively removing it from the public API. It's not ideal but I think we need to keep it for allowing people to move forwards at least until Narayana 8

Reply:
Sorry, I forgot to re setting it to public. I'll update it too. Thanks